test/e2e/facility-helpers.js

```diff
@@ -0,0 +1,267 @@
+const mock = require('./mock-helpers');
+
+/* eslint-disable camelcase */
+const resultsData = {
+  data: [
+    {
+      id: 'vba_343z',
+      type: 'va_facilities',
+      attributes: {
+        unique_id: '343z',
+        name: 'VetSuccess on Campus at Los Angeles City College',
+        facility_type: 'va_benefits_facility',
+        classification: 'VETSUCCESS ON CAMPUS',
+        website: null,
+        lat: 34.08751109,
+        'long': -118.2917626,
```

Comment:
I'm surprised the linter didn't trip over this. Thought it usually complained quoted single-word keys for me.